samtranslator/internal/schema_source/aws_serverless_graphqlapi.py

```diff
@@ -152,7 +152,19 @@ class Cache(BaseModel):
     TransitEncryptionEnabled: Optional[PassThroughProp]
 
 
+class Defaults(BaseModel):
+    # properties common for Resolvers and Functions
+    Runtime: Optional[Runtime]
+    MaxBatchSize: Optional[PassThroughProp]
+    Sync: Optional[Sync]
```

Comment:
Could there be a case where customers would want the defaults to only apply to Resolvers or only to Functions? What would they do in that case?

Reply:
Yes, they will put those properties under `Resolvers` or `Functions` property:
```yaml
Defaults:
  MaxBatchSize: 10 # that's for both Resolvers and Functions
  Resolvers:
    MaxBatchSize: 20 # that's Resolvers only, it overrides '10' from above
  Functions:
    Sync:
      # this Sync setting is only for functions
```

Reply:
Ah that makes sense, so whatever they define in the `Resolvers` will take precedence